### PR TITLE
Adds detection for ComTrade Tesla devices

### PR DESCRIPTION
**Pull request by liviuconcioiu**
refs #5808 - adds detection for ComTrade Tesla devices

### CODE REVIEW

File: Parser/Device/DeviceParserAbstract.php [hunk 1]
```diff
@@ -127,6 +127,7 @@
         'CU' => 'Cube',
         'CY' => 'Coby Kyros',
         'C6' => 'Comio',
+		'C7' => 'ComTrade Tesla',
```
Comment (sgiehl) [inline Parser/Device/DeviceParserAbstract.php]: please use spaces instead of tabs here

**Comment by sgiehl** [comment]
Any reason for naming the brand `ComTrade Tesla`? Doesn't the brand is called `Tesla`, even if it's currently owned by `Comtrade`.

**Comment by liviuconcioiu** [comment]
I tried to do that, but it conflicts with the Tesla car browser...

**Comment by sgiehl** [comment]
Ah forget about that. You're right... So guess it's fine to name it that way.

**Pull request merged**